tests/e2e/performance/test_pvc_creation_performance.py

```diff
@@ -62,24 +64,47 @@ def base_setup(
     @pytest.mark.usefixtures(base_setup.__name__)
     def test_pvc_creation_measurement_performance(self, teardown_factory, pvc_size):
         """
-        Measuring PVC creation time is less than 3 seconds
+        The test measures PVC creation times for sample volumes ( currently samples_num is 3)
+        makes sure create time is not greated than 3 seconds
+        calculates average creation time
+        and that the difference between creation time of each one of the PVCs and the average
+        is not more than 5%
         """
-        log.info('Start creating new PVC')
+        samples_num = 3
+        accepted_diff = 5
+        acceoted_create_time = 3
```

Comment:
typo in `accepted`

Reply:
@keesturam - spelling fixed, thank you!

Reply:
@keesturam ,
Regarding the function that calls a certain operation N ( number of times) times and returns the statistics. 
Such function can be written and this is a matter of our decision. 
This idea has 2 disadvantages:
1) In case such function will only calculate statistics ( but not compare the statistics to the accepted values) - it will "do more job" than required in case one of the measured values is not valid. 
Let's say, if one of the create time values is 5 secs ( invalid) - the current code will through the exception, fail the test and will not calculate any statistics. 
The suggested function will still calculate average/variance/whatever statistics we decide and only then the test will be failed.
2) In case we want such function to calculate statistics ( and be able to fail the test) - this moment if will not be a general function since we will have to pass the accepted values to this function.  

So the question is ( in case I understand things right) is what we decide to do.